alpine-model/src/main/java/alpine/model/ConfigProperty.java

```diff
@@ -66,8 +66,7 @@ public class ConfigProperty implements IConfigProperty, Serializable {
     private String propertyName;
 
     @Persistent
-    @Column(name = "PROPERTYVALUE", length = 1024)
-    @Size(min = 0, max = 1024)
+    @Column(name = "PROPERTYVALUE")
```

Comment:
`jdbcType` will need to be set to `CLOB` in order for DataNucleus to generate `TEXT` columns (unlimited length) for it. See https://github.com/DependencyTrack/dependency-track/blob/adfb827a3718f616eacbdf9f2a71072adf7baef8/src/main/java/org/dependencytrack/model/License.java#L128-L131 for reference.

Reply:
Note that we will need a migration in Dependency-Track for existing databases, since DataNucleus won't do it automatically, like this: https://github.com/DependencyTrack/dependency-track/blob/adfb827a3718f616eacbdf9f2a71072adf7baef8/src/main/java/org/dependencytrack/upgrade/v480/v480Updater.java#L44-L64

Reply:
I am aware of the need for migration. Will do it for all my PRs when the release of them come to DT.